activities/ActivityTemplate/VueJS/js/components/SugarJournal.js

```diff
@@ -71,5 +71,11 @@ Vue.component('sugar-journal', {
 				}, 0);
 			});
 		},
+
+		createEntry: function (data, metadata, callback) {
```

Comment:
Here again, I'm not in favor of using callback. Events or promise will be more coherent.

Reply:
Yes I'll convert this to return a Promise as well.